src/frontend/src/pages/CalendarPage/DesignReviewSummaryModal.tsx

```diff
@@ -11,19 +11,114 @@ import DesignReviewSummaryModalCheckBox from './SummaryComponents/DesignReviewSu
 import DesignReviewSummaryModalButtons from './SummaryComponents/DesignReviewSummaryModalButtons';
 import DesignReviewSummaryModalAttendees from './SummaryComponents/DesignReviewSummaryModalAttendees';
 import { getTeamTypeIcon } from './CalendarComponents/CalendarDayCard';
+import { useEditDesignReview } from '../../hooks/design-reviews.hooks';
+import LoadingIndicator from '../../components/LoadingIndicator';
+import ErrorPage from '../ErrorPage';
+import { useToast } from '../../hooks/toasts.hooks';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
+import { useForm } from 'react-hook-form';
 
 interface DRCSummaryModalProps {
   open: boolean;
   onHide: () => void;
   designReview: DesignReview;
 }
 
+export interface DesignReviewEditProps {
+  dateScheduled: Date;
+  teamTypeId: string;
+  requiredMembers: User[];
+  optionalMembers: User[];
+  isOnline: boolean;
+  isInPerson: boolean;
+  zoomLink: string;
+  location: string;
+  docTemplateLink: string;
+  status: DesignReviewStatus;
+  attendees: number[];
+  meetingTimes: number[];
+}
+
+const schema = yup.object().shape({
+  dateScheduled: yup.date(),
+  teamTypeId: yup.string().required(),
+  requiredMembersIds: yup.array(yup.number()).required(),
+  optionalMemberIds: yup.array(yup.number()).required(),
+  isOnline: yup.boolean(),
+  isInPerson: yup.boolean(),
+  zoomLink: yup
+    .string()
+    .optional()
+    .test('zoom-link', 'Must be a valid zoom link', (value) => value!.includes('zoom.us/')),
+  location: yup.string().optional(),
+  docTemplateLink: yup.string().optional(),
+  // TODO is status cast to string or left as enum?
+  status: yup.string(),
+  attendees: yup.array(yup.number()),
+  meetingTimes: yup.array(yup.number())
```

Comment:
this is a bit weird because we made one big edit endpoint. You should basically only have location, docTemplateLink, and zoomLink in the schema for this form because those are the only 3 fields you can edit from this modal. You can then construct the rest of the edit payload from the current design review values in the designReview we already have in here.

The required/ optional attendee removing should be it's own call to the edit endpoint that happens whenever someone clicks minus, so totally separate from editing those 3 fields (don't worry ab that as much tho, bc ppl can still edit that from the main design review detail page rn)